docker-compose.yml

```diff
@@ -3,18 +3,21 @@ services:
   bmsapp:
     depends_on:
       - bmsmysql
-    networks:
-      - bmsnet
     image: ${BMS_IMG_URI:?err}:${BMS_RELEASE:?err}
     ports:
       - "80:8080"
     restart: always
     environment:
+      - VIRTUAL_HOST=${BMS_SSL_HOST:-bmsdock.leafnode.io}
```

Comment:
I think we don't need these environment properties for non-ssl setup:
`VIRTUAL_HOST
VIRTUAL_PORT
LETSENCRYPT_HOST
LETSENCRYPT_EMAIL`
Right?